src/Resizer.php

```diff
@@ -10,6 +10,9 @@
 
 namespace Contao\Image;
 
+use Contao\Image\Event\ContaoImageEvents;
+use Contao\Image\Event\ResizeImageEvent;
+use Symfony\Component\EventDispatcher\EventDispatcher;
```

Comment:
We should probably use the `Symfony\Component\EventDispatcher\EventDispatcherInterface` here.